source/data/tier1data/schemas/metadata-schemas-tech.rst

```diff
@@ -0,0 +1,533 @@
+##########################################
+Metadata schemas: technical specifications
+##########################################
+
+
+This article describes how the metadata schemas used in the ManGO portal
+are stored and represented, from the folder structure that supports the
+schemas lifecycle to the JSON format that codes the different fields and
+their characteristics.
+
+`Section 1 <#sec-lifecycle-tech>`__ gives a brief overview of the lifecycle
+of a schema and how that is coded into the folder structure and filename
+system. `Section 2 <#sec-json>`__ follows with a technical description
+of the JSON file that represents each version of a specific schema,
+followed by `Section 3 <#sec-items>`__, in which the different kinds of
+fields are described. Finally, `Section 4 <#sec-full>`__ shows an
+example of the JSON file for a draft version.
+
+      If you would like to design your own metadata schema for Tier-1 Data
+      without using the Metadata Schema Manager, you should focus on
+      `Section 3 <#sec-items>`__ and create a JSON file that matches the
+      value of ``properties`` in the main JSON. On upload to Tier-1 Data, you
+      will be able to provide the name and title of your schema, and the
+      versioning will be taken care of in the backend.
```

Comment:
This looks like it shold be a `.. note:`

Reply:
Done